build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -18,16 +18,17 @@
     <PackageReference Include="Microsoft.Build" Version="15.4.8" />
     <PackageReference Include="Microsoft.CSharp" Version="4.0.1" />
     <PackageReference Include="System.Dynamic.Runtime" Version="4.0.11" />
-    <PackageReference Include="System.Reflection.Metadata" Version="1.4.1-beta-24410-02" />
+    <PackageReference Include="System.Reflection.Metadata" Version="1.4.2" />
     <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.1.1" />
     <PackageReference Include="System.Xml.XmlSerializer" Version="4.0.11" />
     <PackageReference Include="WindowsAzure.Storage" Version="7.2.1" />
-    <PackageReference Include="NuGet.CommandLine.XPlat" Version="4.4.0-preview3-4475" />
+    <PackageReference Include="NuGet.CommandLine.XPlat" Version="4.4.0" />
```

Comment:
I think this was just deleted in master. Might make the merge easier to delete it here too.

Reply:
Or maybe that was in a different project.